index.js

```diff
@@ -99,11 +132,17 @@ CachingWriter.prototype._conditionalBuild = function () {
         return rimraf(writer.outputPath);
       }).then(function() {
         fs.mkdirSync(writer.outputPath);
-      });
+      }).finally(function() {
+
+        this.debug('purge output in %dms', new Date() - start);
+        start = new Date();
```

Comment:
Ahh, I see.  We really are tracking two different timings here with `start`.  Thats the bit I missed, thanks for explaining.